editor/editor_vcs_interface.cpp

```diff
@@ -160,6 +223,46 @@ Array EditorVCSInterface::get_file_diff(String p_file_path) {
 	return Array();
 }
 
+Array EditorVCSInterface::get_previous_commits() {
+
+	if (is_addon_ready()) {
+
+		return call("_get_previous_commits");
+	}
+	return Array();
+}
+
+Array EditorVCSInterface::get_branch_list() {
+
+	if (is_addon_ready()) {
+
+		return call("_get_branch_list");
+	}
+	return Array();
+}
+
+bool EditorVCSInterface::checkout_branch(String branch) {
+
+	if (is_addon_ready()) {
+
+		return call("_checkout_branch", branch);
+	}
+
+	return false;
+}
+
+void EditorVCSInterface::pull() {
+	call("_pull");
```

Comment:
The `if (is_addon_ready())` check has not be done here and also in a few functions below, unlike the functions above. Is there a specific reason for that?